pkgs/by-name/ma/managarr/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+  perl,
+}:
+
+let
+  pname = "managarr";
+  version = "0.4.2";
+in
+
+rustPlatform.buildRustPackage {
+  inherit pname version;
+  src = fetchFromGitHub {
+    owner = "Dark-Alex-17";
+    repo = "managarr";
+    rev = "v${version}";
```

Comment:
I'm a bit unsure still about using ref the way I do with the tag directly. It works, as this is how I built the package, but I just see that usually it's something like `/revs/tags/${version}` (nix-update doesn't seem to be able to use the tag attribute yet). If necessary, I can change this.

Reply:
Use gitUpdater instead of nix-update-script, then you only need to specify rev-prefix

Ex: `passthru.updateScript = gitUpdater { rev-prefix = "v"; };`

Reply:
Thanks for the input; for the moment, I've left it at nix-update-script in order not to change too much, and the only example of a Rust package using autoupdate I could find quickly was kanidm, which uses it to specify some options. I assume the bot runs update scripts on all packages automatically anyways so stating this here isn't strictly necessary.

Also it's a bit opaque to me how the hashes are updated, I can run the update scripts, but it will only update the versions here, but hashes remain unchanged, so there needs to be something happening in the background that isn't explained where I have looked at yet.